src/schema/generic.json

```diff
@@ -4,6 +4,18 @@
     "inputs": {},
     "outputs": {},
     "types": {
+        "Event": {
+            "type": "object",
+            "description": "An object describing an specific event emitted by automation job. The exact event types and data depend on the specific service.",
+            "required": ["eventType"],
+            "properties": {
+                "eventType": {
```

Comment:
We don't typically repeat the parent key's name inside their children. E.g.:

```json
{
  "hotel": {
    "name": {...},
    "address": {...},
  }
}

vs

{
  "hotel": {
    "hotelName": {...},
    "hotelAddress": {...},
  }
}
```

Reply:
Agreed, let's make it consistent here